### PR TITLE
formatting 4/n: Trainer

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

deploy formatting for Trainer

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

**Comment by akihironitta** [comment]
Following the convention in PyTorch, we could probably ignore both W503 and W504. https://github.com/pytorch/pytorch/blob/721ba97eb6322cdcba2eff6cf9149f462f7a1989/.flake8#L7
```
Line break occurred before a binary operator (W503)
Line break occurred after a binary operator (W504)
```

**Review by ananthsub — APPROVED** [review]

File: pytorch_lightning/trainer/deprecated_api.py [hunk 1]
```diff
@@ -119,14 +115,16 @@ def use_single_gpu(self) -> bool:
             "Internal: `use_single_gpu` is deprecated in v1.2 and will be removed in v1.4.", DeprecationWarning
         )
         # todo, limiting to exclude DDP2 is not clear but it comes from connectors...
-        return (self._device_type and self._device_type == DeviceType.GPU
-                and self.num_gpus == 1
-                and self._distrib_type not in (DistributedType.DDP2, ))
+        return (
+            self._device_type and self._device_type == DeviceType.GPU and self.num_gpus == 1
+            and self._distrib_type not in (DistributedType.DDP2, )
```
Comment (rohitgr7) [inline pytorch_lightning/trainer/deprecated_api.py:120]:
```suggestion
            and self._distrib_type != DistributedType.DDP2
```

File: pytorch_lightning/trainer/logging.py [hunk 2]
```diff
@@ -66,13 +66,12 @@ def process_dict_result(self, output, train=False):
...
-                        Please use self.log(...) inside the lightningModule instead.
-
-                        # log on a step or aggregate epoch metric to the logger and/or progress bar
-                        # (inside LightningModule)
-                        self.log('train_loss', loss, on_step=True, on_epoch=True, prog_bar=True)
-                    """)
+                        f"The {{{k}:dict keyword}} was deprecated in 0.9.1 and will be removed in 1.0.0"
+                        " Please use self.log(...) inside the lightningModule instead."
+                        " # log on a step or aggregate epoch metric to the logger and/or progress bar"
+                        " (inside LightningModule)\n"
```
Comment (rohitgr7) [inline pytorch_lightning/trainer/logging.py:72]: is this correct?

**Review by rohitgr7 — APPROVED** [review]

File: pytorch_lightning/trainer/data_loading.py [hunk 3]
```diff
@@ -37,12 +37,12 @@ class TrainerDataLoadingMixin(ABC):
     # this is just a summary on variables used in this abstract class,
     #  the proper values/initialisation should be done in child class
     global_rank: int
-    shown_warnings: ...
+    shown_warnings:...
```
Comment (rohitgr7) [inline pytorch_lightning/trainer/data_loading.py:40]: also, pep8 is having some issues with this: https://github.com/PyTorchLightning/pytorch-lightning/pull/5720#issuecomment-770295218

Comment (Borda) [reply pytorch_lightning/trainer/data_loading.py:40]: can you report it on YAPF?

File: pytorch_lightning/trainer/logging.py [hunk 4]
```diff
@@ -66,13 +66,12 @@ def process_dict_result(self, output, train=False):
...
                     m = inspect.cleandoc(
-                        f"""The {{{k}:dict keyword}} was deprecated in 0.9.1 and will be removed in 1.0.0
-                        Please use self.log(...) inside the lightningModule instead.
-
-                        # log on a step or aggregate epoch metric to the logger and/or progress bar
-                        # (inside LightningModule)
-                        self.log('train_loss', loss, on_step=True, on_epoch=True, prog_bar=True)
-                    """)
+                        f"The {{{k}:dict keyword}} was deprecated in 0.9.1 and will be removed in 1.0.0"
+                        " Please use self.log(...) inside the lightningModule instead."
```
Comment (Borda) [inline pytorch_lightning/trainer/logging.py:70]:
```suggestion
                        f"The {{{k}:dict keyword}} was deprecated in 0.9.1 and will be removed in 1.0.0\n"
                        " Please use self.log(...) inside the lightningModule instead.\n"
```

File: pytorch_lightning/trainer/logging.py (hunk 2 above)
Comment (Borda) [reply pytorch_lightning/trainer/logging.py:72]: What do you mean?

File: pytorch_lightning/trainer/data_loading.py (hunk 3 above)
Comment (Borda) [reply pytorch_lightning/trainer/data_loading.py:40]: I do not see flake8 complaining about this change...

File: pytorch_lightning/trainer/logging.py (hunk 2 above)
Comment (rohitgr7) [reply pytorch_lightning/trainer/logging.py]: I thought
```
# log on a step or aggregate epoch metric to the logger and/or progress bar
```
this is a comment here and now added to the string. BTW as per TODO this should have been removed in 1.0?

Comment (Borda) [reply pytorch_lightning/trainer/logging.py]: well the comment was there even before as before it was text block so including it too

**Review by s-rog — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Pull request merged**